modules/installation-configuration-parameters.adoc

```diff
@@ -1116,8 +1116,8 @@ within link:https://azure.microsoft.com/en-us/global-infrastructure/regions[a re
 you are using user-defined routing, you must have pre-existing networking
 available where the outbound routing has already been configured prior to
 installing a cluster. The installation program is not responsible for
-configuring user-defined routing.
-|`LoadBalancer` or `UserDefinedRouting`. The default is `LoadBalancer`.
+configuring user-defined routing. If you specify the `NatGateway` routing strategy, the installation program will only create one NAT gateway. If you specify the `NatGateway` routing strategy, your account must have the `Microsoft.Network/natGateways/read` and `Microsoft.Network/natGateways/write` permissions. The `NatGateway` outbound routing strategy is a Technology Preview feature in {product-title} {product-version}.
```

Comment:
```suggestion
configuring user-defined routing. If you specify the `NatGateway` routing strategy, the installation program will create only one NAT gateway. If you specify the `NatGateway` routing strategy, your account must have the `Microsoft.Network/natGateways/read` and `Microsoft.Network/natGateways/write` permissions. The `NatGateway` outbound routing strategy is a Technology Preview feature in {product-title} {product-version}.
```

Reply:
Do we need the Technology Preview snippet to be included, like in a footnote, or is it OK to mention a TP feature without having the snippet?

Reply:
Let me try including the snippet in the table and see if it messes with the formatting.

Reply:
Yeah, it might - good luck <fingers crossed>

Reply:
Looks OK in my local build - PTAL once travis and netlify update in a few minutes. Thanks

Reply:
It looks probably as good as it's going to, which is to say not awesome, but it's not breaking the table at least. :)